ui/app/components/clients/dashboard.js

```diff
@@ -41,15 +39,6 @@ export default class Dashboard extends Component {
     return this.args.model.activity && this.args.model.activity.total;
   }
 
-  // Show namespace graph only if we have more than 1
```

Comment:
carry over from copying the history tab. not needed in the new designs.

Reply:
Yes - each component will manage it displays the graph, or shows an empty state view